dbt_copilot_helper/templates/addons/env/opensearch.yml

```diff
@@ -86,10 +85,9 @@ Resources:
       ToPort: 443
       SourceSecurityGroupId: !Ref EnvironmentSecurityGroup
 
-  # Single opensearch instance
+  # Opensearch
```

Comment:
This comment seems a bit redundant to me, as this is the OpenSearch addon template.

```suggestion
```